state/factory/workingset.go

```diff
@@ -128,9 +127,10 @@ func (ws *workingSet) runActions(
 
 func withActionCtx(ctx context.Context, selp action.SealedEnvelope) (context.Context, error) {
 	var actionCtx protocol.ActionCtx
-	caller, err := address.FromBytes(selp.SrcPubkey().Hash())
-	if err != nil {
-		return nil, err
+	var err error
```

Comment:
no need to define this?

Reply:
I guess its needed for next line, actionCtx.ActionHash, err := selp.Hash()
As actionCtx variable is already defined, err also needs to be defined. Earlier it was working because err was getting defined in caller, err := address.FromBytes(selp.SrcPubkey().Hash()) which is removed now.

Reply:
i see, thx